addons/utm/tests/test_utm.py

```diff
@@ -13,13 +16,18 @@ def test_find_or_create_record(self):
         }])
```

Comment:
Would add a test to demonstrate the actual value of the same, e.g. `self.assertEqual(source_1.name, 'source 1')` and maybe add a comment saying they should always be created in lowercase so when we come back to it it's clear it was intentional.